scripts/pull_github_pr.sh

```diff
@@ -17,7 +17,16 @@ if [[ ( -z "$GITHUB_LOGIN" || -z "$GITHUB_TOKEN" ) && -f "$gh_hosts" ]]; then
 	GITHUB_TOKEN=$(awk '/oauth_token:/ { print $2 }' "$gh_hosts")
 fi
 
-if [[ $# != 1 ]]; then
+if [[ -z "$JENKINS_USERNAME" || -z "$JENKINS_API_TOKEN" ]]; then
+  echo """
+    JENKINS_USERNAME or JENKINS_API_TOKEN is missing from enva
+    To create a TOKEN, browse to https://jenkins.scylladb.com, then click on your username (upper right corner) and configure. Click on Add new token
```

Comment:
The sentence just ends in the middle. Needs a final point, and also maybe say something like "and set the JENKINS_USERNAME and JENKINS_API_TOKEN environment variables accordingly"?